code/modules/plumbing/plumbers/plumbing_buffer.dm

```diff
@@ -31,11 +31,11 @@
 /obj/machinery/plumbing/buffer/proc/on_reagent_change()
 	if(!buffer_net)
 		return
-	if(reagents.total_volume >= activation_volume && mode == UNREADY)
+	if(reagents.total_volume + CHEMICAL_QUANTISATION_LEVEL >= activation_volume && mode == UNREADY)
 		mode = IDLE
 		buffer_net.check_active()
 
-	else if(reagents.total_volume < activation_volume && mode != UNREADY)
+	else if(reagents.total_volume + CHEMICAL_QUANTISATION_LEVEL < activation_volume && mode != UNREADY)
```

Comment:
We're checking if it's lower so should probably minus

Reply:
I'm not 100% on if it should be. The change I made in `reaction_chamber.dm` was to do `+` even though it was less than, as we were making sure if we're close enough to the total volume to stop.

I assume it's similar here? We only mark it as `UNREADY` if it's volume is too small? So adding makes sense to me.

Reply:
I think you’re right in that it should be inversed. It’s been a while since I’ve touched the floating point errors

Reply:
The more I think about it, the more I think they're both add. We're basically lying about the Volume in both scenarios, saying it's fractionally bigger than it really is. Imagine if we'd wrapped the `total_volume` in a property/function that returned `total_volume + CHEMICAL_QUANTISATION_LEVEL `, that's how I see it. 

Are you happy to leave them both as `+`, @Time-Green? I don't think we should ever deduct from the volume, only add.

Reply:
Sure, I trust your judgement this

Reply:
Okay, cool. I'll keep an eye on it but I believe this is correct.